third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -1006,8 +932,12 @@ struct AsyncCommitGroupOpConversion
     PTXBuilder ptxBuilder;
     ptxBuilder.create<>("cp.async.commit_group")->operator()();
     ptxBuilder.launch(rewriter, op.getLoc(), void_ty(op.getContext()));
-    // Safe to remove the op since it doesn't have any return value.
-    rewriter.eraseOp(op);
+
+    // Drop the result token.
+    Value zero = rewriter.create<LLVM::ConstantOp>(
```

Comment:
Isn't the return value an asynchronous token?

Reply:
it is, but those don't exist anymore at LLVM level, they are handled by side-effects and op ordering only